geopyspark/geotrellis/catalog.py

```diff
@@ -415,8 +423,9 @@ def write(uri,
             method used to orginize the saved data. Depending on the type of data within the layer,
             only certain methods are available. Can either be a string or a ``IndexingMethod``
             attribute.  The default method used is, ``IndexingMethod.ZORDER``.
-        time_unit (str, optional): Which time unit should be used when saving spatial-temporal data.
-            While this is set to None as default, it must be set if saving spatial-temporal data.
+        time_unit (str or :class:`~geopyspark.geotrellis.constants.TimeUnit`, optional): Which time
+            unit should be used when saving spatial-temporal data.  While this is set to ``None``
+            as default, it must be set if saving spatial-temporal data.
```

Comment:
Noticed that this doc string is misleading. This controls temporal resolution of the index. Approximately at what time interval the records are spaced temporally. Is there an image for every day, hour, minute?